hw/ip/rstmgr/data/rstmgr.hjson

```diff
@@ -0,0 +1,137 @@
+// Copyright lowRISC contributors.
```

Comment:
I guess for bronze we might as well go ahead and keep this, but with so few
registers it is tempting to have them join with like PMU or something and not
pay the TLUL cost.